deploy/roles/r-core/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Ensure umask is correct
+  lineinfile:
+    dest: "~/.bashrc"
+    line: umask 0022
+    regexp: (.*)umask(.*)
+    state: present
```

Comment:
@lel99999 This should have the `tags: r-core` parameter, or else it might not get run.